kivy/logger.py

```diff
@@ -115,8 +116,8 @@ def purge_logs(self, directory):
         if randint(0, 20) != 0:
             return
 
-        # Use config ?
-        maxfiles = 100
+        from kivy.config import Config
+        maxfiles = Config.getint('kivy', 'log_maxfiles') - 1
```

Comment:
I myself feel that the `log_maxfiles` should represent a number for the real state _before_ actually creating a new log i.e. keeping the variable kind of "local" to the `purge_logs()`, independent on the rest of the system. Imagine a case when someone wants to purge the logs manually(`purge_logs()` isn't a private method), this could lead to a confusion as for "why is there a file missing?".

Imagine a value from config equal to `10` and a total number of files equal to `100`. In case when `maxfiles = <config>` this is the output:
* total: 100
* erased: 90
* remaining: 10

in case when `maxfiles = <config> - 1` this is the output:
* total: 100
* erased: 91
* remaining: 9